front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -49,10 +49,28 @@ export class AnalysisPage extends Component {
             data-testid="analysis-page-main"
           >
             <div className="flex gap-y-10 w-full flex-col h-full">
-              <div className= "w-full bg-gray-100 pb-2">
+              <div className= "flow-root w-full bg-gray-100 pb-2">
+                {
+                    !this.state.showIntentMenu &&
+                    <nav>
+                      <button
+                        className="float-left m-auto px-5 py-3"
```

Comment:
nit: it is best to avoid float. Try justify-start and making the parent flex (generally the hierarchy of CSS goes flex > margin > float).

Reply:
To add, we don't want to use float here largely because float is meant to be eventually cleared. So using it for permanent styling is bad practice.